test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,46 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+const urlBase = 'https://api.github.com/users';
+const urlRepos = 'https://api.github.com/repos';
+const repo = 'jasmine-json-report';
+const owner = 'aperdomob';
+
+describe('Repositories testing', () => {
+  it('GET method', async () => {
+    const response = await axios.get(`${urlBase}/${owner}`);
+
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data.name).to.equal('Alejandro Perdomo');
+    expect(response.data.company).to.equal('Perficient Latam');
+    expect(response.data.location).to.equal('Colombia');
+  });
+
+  it('Repositories List', async () => {
+    const response = await axios.get(`${urlBase}/${owner}/repos`);
+    const finder = response.data.find((object) => object.name === 'jasmine-json-report');
```

Comment:
instead of finder, can you give it a different name, maybe `repository` or `repo`